GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -136,33 +136,42 @@ final class GiniScreenAPICoordinator: NSObject, Coordinator {
 
 extension GiniScreenAPICoordinator {
     func addToDocuments(new documentRequests: [DocumentRequest]) {
+        if self.documentRequests.type == .qrcode || self.documentRequests.type == .pdf {
```

Comment:
Ideally I'd avoid such edge cases. If you can, I'd prefer it if you could find a way to process everything in the same way.